spec/system/import_and_show_work_spec.rb

```diff
@@ -85,7 +85,7 @@
       expect(page).to have_content work.title.first
       expect(page).to have_content work.description.first
       expect(page).to have_content work.normalized_date.first
-      expect(page).to have_content work.resource_type.first
+      # expect(page).to have_content work.resource_type.first
```

Comment:
Did you mean to comment this out? Shouldn't we test that the resource type is importing correctly?

Reply:
That line was testing the search results, where we removed it. Line 58 tests it on the show page. I've deleted 88 instead of commenting it out, though.